Makefile

```diff
@@ -1,6 +1,6 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= hypersds-operator:latest
```

Comment:
It seems that registry information should be included for `docker-push`.

Reply:
Added on the todo list for the next PR